app/src/main/java/org/exoplatform/fragment/WebViewFragment.java

```diff
@@ -89,29 +85,19 @@ public void onCreate(Bundle savedInstanceState) {
   @Override
   public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
     RelativeLayout layout = (RelativeLayout) inflater.inflate(R.layout.fragment_web_view, container, false);
-    mPageTitle = (TextView) layout.findViewById(R.id.WebViewFragment_PageTitle);
-    mCloseButton = (ImageButton) layout.findViewById(R.id.WebViewFragment_CloseButton);
-    mCloseButton.setOnClickListener(new View.OnClickListener() {
-      @Override
-      public void onClick(View v) {
-        if (mListener != null)
-          mListener.onCloseWebViewFragment();
-      }
-    });
     mWebView = (WebView) layout.findViewById(R.id.WebViewFragment_WebView);
     mWebView.setWebViewClient(new WebViewClient());
     mWebView.getSettings().setJavaScriptEnabled(true);
     mWebView.getSettings().setDomStorageEnabled(true);
     mWebView.getSettings().setUserAgentString("eXo/" + BuildConfig.VERSION_NAME + " (Android)");
     mWebView.getSettings().setLayoutAlgorithm(WebSettings.LayoutAlgorithm.NORMAL);
     mWebView.getSettings().setBuiltInZoomControls(true);
-    mWebView.getSettings().setDisplayZoomControls(true);
+    mWebView.getSettings().setDisplayZoomControls(false);
```

Comment:
why disable the Display zoom controls ?

Reply:
setDisplayZoomControls(false) is used  to hide the zoom buttons when zomming in.